Backend/dhall-configs/dev/rider-dashboard.dhall

```diff
@@ -87,6 +87,7 @@ in  { esqDBCfg
     , encTools
     , exotelToken = ""
     , dataServers = [ appBackend, appBackendManagement ]
+    , merchantUserAccountNumber = +5
```

Comment:
Why is this extra config added ?

Reply:
it was missed in `main`. Now after rebase this config is already there